src/views/ont/OntPlates.vue

```diff
@@ -38,37 +47,40 @@ export default {
   name: 'OntPlates',
   data () {
     return { 
-      fields: [ 
-        {key: 'id', label: 'ID', sortable: true},
-        {key: 'barcode', label: 'Barcode', sortable: true},
-        {key: 'createdAt', label: 'Created at', sortable: true},
-        {key: 'show_details', label: 'Show Details'}
-      ],
-      sortBy: 'createdAt',
-      sortDesc: true,
+      fields: [ 'id','barcode','createdAt','show_details'],
+      perPage: 5,
+      currentPage: 1,
+      totalRows: 0
     }
   },
   components: {
     OntPlate,
     Alert
   },
   mixins: [Helper],
-  apollo: {
-    plates: {
-      query: PLATES_ALL_QUERY
-    }
-  },
   methods: {
     alert(message, type) {
       this.showAlert(message, type)
     },
-    refetchPlates() {
-      this.$apollo.queries.plates.refetch()
+    getPlates(ctx, callback) {
+      this.$apollo.query({
+        query: PLATES_ALL_QUERY,
+        variables: {
+          pageNum: ctx.currentPage,
+          pageSize: ctx.perPage
+        },
+        fetchPolicy: 'no-cache'
```

Comment:
More a query than a code review comment. Would we be able to cache this in future?

Reply:
Yes I think so